playbook/app/pb_kits/playbook/pb_button/_button.jsx

```diff
@@ -106,6 +106,7 @@ const Button = (props: ButtonPropTypes) => {
           className={css}
           href={link}
           id={id}
+          rel="noreferrer"
```

Comment:
is this used for SEO?

Reply:
It is, however, it is also part of the linter settings. We may choose later to remove this however it does serve 3rd party consumers well.